navbar-service.js

```diff
@@ -28,6 +28,9 @@ function factory(config) {
   // exposes menus for configuration
   service.menus = {};
 
+  service.brandOnly = false;
```

Comment:
Hmm, I'm not sure this is the best name for this. Especially if we want to enable more fine-grained control, in the future, of making other parts of the navbar display. We've got the brand, the menu, and the toolbar. Maybe we want to let people call out those items that they'd like to display. So if you just wanted brand you'd set a display var to `'brand'` or something like that.